payments-core/build.gradle

```diff
@@ -111,6 +112,7 @@ android {
 
     lintOptions {
         enable "Interoperability"
+        disable "CoroutineCreationDuringComposition"
```

Comment:
This test is [causing a crash](https://github.com/stripe/stripe-android/pull/3840/checks?check_run_id=2908422002) (it doesn't look like we're failing the lint, but the checker is crashing)